python_scripts/02_basic_preprocessing.py

```diff
@@ -102,7 +84,8 @@
 
 # "i" denotes integer type, "f" denotes float type
 numerical_columns = [
```

Comment:
We might want to use the `make_column_selector` which might more explicit in the construction since we specify better the dtype.

```python
from sklearn.compose import make_column_selector as selector

numerical_columns_selector = selector(dtype_include=["int", "float"])
numerical_columns = numerical_columns_selector(data)
```

Reply:
I created https://github.com/INRIA/scikit-learn-mooc/issues/20 PR more than welcome (this is quite nice to create an issue from a comment by the way I did not know it was possible and that easy)